src/test/java/core/basesyntax/model/UserTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+
+class UserTest {
```

Comment:
It's redundant test class. As I see you are trying to test if Java methods work well, there is no logic to test.